tests/unittests/RecommendationSystemTest.cpp

```diff
@@ -92,6 +92,12 @@ llvm::cl::opt<bool> useSymmetricRowwiseQuantFCOpt(
     llvm::cl::desc(
         "Whether to use Symmetric quantization with FCs. Default is false."),
     llvm::cl::Optional, llvm::cl::init(false), llvm::cl::cat(recSysTestCat));
+
+llvm::cl::opt<std::string> traceFilePrefix(
```

Comment:
It seems like you use `traceFilePrefix` semantically as a directory name. If this is correct, why not just call this option `traceDir` or something like that?